engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -271,20 +252,16 @@ private void preRenderUpdate(RenderingStage renderingStage) {
         resetStats();
 
         currentRenderingStage = renderingStage;
-        if (currentRenderingStage == RenderingStage.MONO || currentRenderingStage == RenderingStage.LEFT_EYE) {
-            isFirstRenderingStageForCurrentFrame = true;
-        } else {
-            isFirstRenderingStageForCurrentFrame = false;
-        }
+        isFirstRenderingStageForCurrentFrame = currentRenderingStage == RenderingStage.MONO || currentRenderingStage == RenderingStage.LEFT_EYE;
```

Comment:
Hmmmm... of course here the code it's syntactically equivalent to the previous version. In fact, I think originally the code was similar to what you suggest. However, I remember changing it into an if/else block to make it more readable given how important ifFirstRenderingStageForCurrentFrame is. So, I'd prefer to keep it as it is for the time being. Eventually the whole thing will go away as we deal with a Stereoscopic setup through RenderNodes and the pipeline.